charts/dbildungscloud-iam-keycloak/templates/deployment.yaml

```diff
@@ -1,102 +1,101 @@
+              path: {{ .Values.startupProbe.httpRelativePath }}
+            failureThreshold: {{ .Values.startupProbe.failureThreshold }}
+            periodSeconds: {{ .Values.startupProbe.periodSeconds }}
           livenessProbe:
             httpGet:
-              port: web
-              path: "/health/live"
+              port: {{ .Values.service.name }}
+              path:  {{ .Values.livenessProbe.path }}
           readinessProbe:
             httpGet:
-              port: web
-              path: "/health/ready"
+              port: {{ .Values.service.name }}
+              path: {{ .Values.readinessProbe.path }}
           env:
             - name: KEYCLOAK_ADMIN
               valueFrom:
                 secretKeyRef:
-                  name: {{ .Values.secrets.name }}
+                  name: {{ .Values.secrets.keycloaksecretname }}
                   key:  admin-user
             - name: KEYCLOAK_ADMIN_PASSWORD
               valueFrom:
                 secretKeyRef:
-                  name: {{ .Values.secrets.name }}
-                  key:  admin-password
+                  name: {{ .Values.secrets.keycloaksecretname }}
+                  key: admin-password
             - name: KC_PROXY
               value: "edge"
             - name: DB_HOST
               valueFrom:
                 secretKeyRef:
-                  name: {{ .Values.secrets.name }}
+                  name: {{ .Values.secrets.keycloaksecretname }}
                   key: db-host
             - name: DB_NAME
               valueFrom:
                 configMapKeyRef:
-                  name: {{ .Values.configmap.name }}
+                  name: {{ template "common.names.configmap" . }}
                   key: db-name
             - name: KC_DB_USERNAME
               valueFrom:
                 secretKeyRef:
-                  name: {{ .Values.secrets.name }}
+                  name: {{ .Values.secrets.keycloaksecretname }}
                   key: db-username
             - name: KC_DB_PASSWORD
               valueFrom:
                 secretKeyRef:
-                  name: {{ .Values.secrets.name }}
+                  name: {{ .Values.secrets.keycloaksecretname }}
                   key: db-password
             - name: KC_DB_URL
               value: "jdbc:postgresql://$(DB_HOST)/$(DB_NAME)"
             - name: KC_CLIENT_SECRET
               valueFrom:
                 secretKeyRef:
-                  name: {{ .Values.secrets.name }}
+                  name: {{ .Values.secrets.keycloaksecretname }}
                   key: keycloak-clientSecret
             - name: KC_ADMIN_SECRET
               valueFrom:
                 secretKeyRef:
-                  name: {{ .Values.secrets.name }}
+                  name: {{ .Values.secrets.keycloaksecretname }}
                   key: keycloak-adminSecret
             - name: KC_ROOT_URL
-              value: "https://{{.Values.frontendHostname}}"
+              value: "https://{{ .Values.ingress.hostname }}"
```

Comment:
I think ingress.hostname is not correkt here, frontendHostname is not equal keycloakhostname

Reply:
No it isn't. Sorry I missed that. This must be frontendHostname it gets injected into the Keycloak-Config